electrum/gui/qt/console.py

```diff
@@ -86,7 +89,9 @@ def updateNamespace(self, namespace):
         self.namespace.update(namespace)
 
     def showMessage(self, message):
-        curr_line = self.getCommand()
+        doc = self.document()
+        curr_line = doc.findBlockByLineNumber(doc.lineCount() - 1).text()
+        curr_line = curr_line[len(sys.ps1):]
```

Comment:
I would prefer parameterising `getCommand()` to optionally not do the rstrip instead of duplicating it here.
I suggest `def getCommand(self, *, strip=True)`

Reply:
I was thinking of doing something like that at first but I also wonder if rstrip() is really useful. I will do it like you ask.